### ISSUE TITLE
Upon installing clear-live-desktop, press ALT+{TDB} to open {TDB}

### ISSUE THREAD

**Issue by ganweehuang** [clearlinux/distribution#662]
Description
TC-gdm-004: Upon installing clear-live-desktop, press ALT+{TDB} to open {TDB}

Environment:
 Clear Linux Desktop

Reproduce step:
1. Install clear Linux 29120
2. Press ALT+{TDB}

Expected result:
Keyboard shortcut (ALT+TAB) is available & able to run ( will open next in line apps)

Current result
No Keyboard function for {TDB} there fore ALT + {TDB} cannot be execute

**Comment by ganweehuang** [comment]
Environment:
•Clear Linux OS version: [cat /usr/lib/os-release]
 NAME="Clear Linux OS"
 VERSION=1
 ID=clear-linux-os
 ID_LIKE=clear-linux-os
 VERSION_ID=29180
 PRETTY_NAME="Clear Linux OS"
 ANSI_COLOR="1;35"
 HOME_URL="https://clearlinux.org"
 SUPPORT_URL="https://clearlinux.org"
 BUG_REPORT_URL="mailto:dev@lists.clearlinux.org"
 PRIVACY_POLICY_URL="http://www.intel.com/privacy"

**Comment by ahkok** [comment]
I have no idea what `TDB` is. This is not a universally understood code. Please describe it.

**Comment by ahkok** [comment]
@ganweehuang It's nice you edited the title, but can you please explain what `{TDB}` is?

**Comment by AthenasJimenez** [comment]
@ganweehuang which application are you using? 

Recently, I saw a similar behavior on gnome-terminal where the menu bar was replaced by a header bar (this is a gnome general change on design). In the new header bar instead of having the usual menus like "File", "Edit", etc. We have a primary menu (square button with 3 dots) that can be open from the keyboard using F10. From there, the regular shortcuts work as usual. 

i.e ALT + W to open a new window.
